package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cactbot",
-  "version": "0.33.4",
-  "releaseSummary": "Interphos, Jeuno, Cloud Chaotic, more FRU",
+  "version": "0.33.5",
+  "releaseSummary": "KO 7.05, more FRU",
   "releaseInDraft": "false",
```

Comment:
Should we change `releaseInDraft` to true?  It might be helpful to add something to the release notes about the 3 new config options for FRU.  Feel free to disregard though if you don't think it's worthwhile.

Reply:
If you want to add some additional info about new config options, feel free to PR a bump with what you want and I'll just take this one down.